<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AKS Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;black&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clearall">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(250, 10, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(250, 10, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 10, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_multiply">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 150, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 150, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_decimal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_percentage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white; /* No quotes around property values */
    color: rgba(0, 0, 0, 0.5);
    border-radius: 10px; /* Numeric value without quotes */
}

QPushButton:hover {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: rgba(0, 0, 0, 1);
    border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_divide">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 150, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clearone">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 100, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(255, 100, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 100, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_subtract">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 150, 0, 0.7);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(255, 150, 0, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_result">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(18, 255, 50, 0.6);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:hover {
    background-color: rgba(18, 255, 50, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}

QPushButton:pressed {
    background-color: rgba(18, 255, 50, 1);
    color: black; /* No quotes around &quot;black&quot; */
    border-radius: 10px; /* No quotes around numeric value */
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputbox_expression">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>381</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:&quot;white&quot;; border-radius: &quot;20px&quot;; color: &quot;black&quot;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:&quot;white&quot;; border-radius: &quot;20px&quot;; color: rgb(0,120,255)</string>
    </property>
    <property name="text">
     <string>kjjfgasdfg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
